src/jvm/test/resources/expected_logs/suspend_trace_reporting_jdk11.txt

```diff
@@ -29,12 +29,12 @@ The following interleaving leads to the error:
 |         isReusable(): false at CancellableContinuationImpl.getResult(CancellableContinuationImpl.kt:297)                                                                                  |                                                                                                                                                                      |
 |         trySuspend(): true at CancellableContinuationImpl.getResult(CancellableContinuationImpl.kt:300)                                                                                   |                                                                                                                                                                      |
 |         getParentHandle(): null at CancellableContinuationImpl.getResult(CancellableContinuationImpl.kt:310)                                                                              |                                                                                                                                                                      |
-|         installParentHandle(): ChildContinuation#1 at CancellableContinuationImpl.getResult(CancellableContinuationImpl.kt:311)                                                           |                                                                                                                                                                      |
-|         switch (reason: coroutine is suspended)                                                                                                                                           |                                                                                                                                                                      |
+|         switch                                                                                                                                                                            |                                                                                                                                                                      |
```

Comment:
The switch reason seems to miss in the trace (other txts as well). Is this intended?

Reply:
I think in this particular test, what has happened is that switch point position changed. 
The `installParentHandle()` method call now happens after the thread switch. 

Before, the thread switch was forced due to coroutine suspension.
Now it is just a strategy switch (i.e., thread switch inserted by the strategy itself) --- these thread switches do not have reason string.